tests/DockerfileDocs

```diff
@@ -0,0 +1,18 @@
+FROM debian:8
+
+RUN apt-get update && \
+    apt-get install -y xsltproc ca-certificates libxml2-utils build-essential git curl python2.7 && \
+		rm -rf /var/lib/apt/lists/*
+
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-$(dpkg --print-architecture)" \
+    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-$(dpkg --print-architecture).asc" \
+    && gpg --verify /usr/local/bin/gosu.asc \
+    && rm /usr/local/bin/gosu.asc \
+    && chmod +x /usr/local/bin/gosu
+
+COPY entrypoint.sh /usr/local/bin/entrypoint.sh
```

Comment:
Is the entrypoint used? There's only `CMD` invoked later on.

Reply:
Great catch. It should use the entrypoint, as the user switching happens there